ROMFS/px4fmu_common/mixers/AERTW.main.mix

```diff
@@ -0,0 +1,95 @@
+Aileron/rudder/elevator/throttle/wheel mixer for PX4FMU
```

Comment:
This title should read: 
Aileron/elevator/rudder/throttle/wheel mixer for PX4FMU